pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
+	cmd.Flags().String("cert-file", "", "Provide certificate contents for a secure route. Required if the route uses edge or reencryption termination.")
+	cmd.Flags().String("key-file", "", "Provide key contents for a secure route. Required if the route uses edge or re-encryption termination.")
+	cmd.Flags().String("ca-cert-file", "", "Provide cert authority certificate contents for a secure route. Optional for edge termination, required for reencrypt.")
+	cmd.Flags().String("dest-ca-cert-file", "", "Provide the contents of the ca certificate of the final destination. Required for reencryption termination routes.")
```

Comment:
'Required if the route uses reencryption termination'  otherwise this last sentence is inconsistent with the last sentences on the other descriptions.